src/components/CodeBlock.tsx

```diff
@@ -66,15 +66,12 @@ export const CodeBlock: React.FC<CodeBlockProps> = ({
   preview
 }) => {
   const language = className?.replace(/language-/, '') as Language
+  const code = `<>${children.trim()}</>`
 
   if (live) {
     return (
-      <LiveProvider
-        code={children.trim()}
-        scope={Components}
-        theme={theme}
-      >
-        <StyledLivePreview layout={center && 'center'} />
+      <LiveProvider code={code} scope={Components} theme={theme}>
+        <StyledLivePreview center={center} />
```

Comment:
Wouldn't this work since it is a truthy variant?
```suggestion
        <StyledLivePreview center />
```

Reply:
This could be `true` or `undefined` (if it's not used) so we need this logic.